src/components/ProductBox/ProductBox.js

```diff
@@ -1,54 +1,56 @@
-import { useState } from 'react';
+import {useState} from 'react';
 import {
-  Image,
-  Heading,
-  Alert,
-  AlertIcon,
-  AlertTitle,
-  AlertDescription,
-  Box,
-  CloseButton
+    Image,
+    Heading,
+    Alert,
+    AlertIcon,
+    AlertTitle,
+    AlertDescription,
+    Box,
+    CloseButton
 } from '@chakra-ui/react';
-import { AddToCart } from '../AddToCart';
+
+import {AddToCart} from '../AddToCart';
 
 import './styles.scss';
 
-const ProductBox = ({ id, image, name, price }) => {
-  const [addedToCart, setAddedToCart] = useState(false);
-  const handleAddToCartClick = (product, qty) => {
-    setAddedToCart(true);
-  };
-  return (
-    <Box className="product-box" maxW="sm" overflow="hidden">
-      {addedToCart && (
-        <Alert status="success">
-          <AlertIcon />
-          <Box flex="1">
-            <AlertTitle>Success!</AlertTitle>
-            <AlertDescription display="block">
-              Product {name} added to cart
-            </AlertDescription>
-          </Box>
-          <CloseButton
-            onClick={() => setAddedToCart(false)}
-            position="absolute"
-            right="8px"
-            top="8px"
-          />
-        </Alert>
-      )}
+const ProductBox = ({id, image, name, price}) => {
+    const [addedToCart, setAddedToCart] = useState(false);
+    const handleAddToCartClick = (product, qty) => {
+        setAddedToCart(true);
+    };
+
+    return (
+        <Box className="product-box" maxW="sm" overflow="hidden">
+            {addedToCart && (
+                <Alert status="success">
+                    <AlertIcon/>
+                    <Box flex="1">
+                        <AlertTitle>Success!</AlertTitle>
+                        <AlertDescription display="block">
+                            Product {name} added to cart
+                        </AlertDescription>
+                    </Box>
+                    <CloseButton
+                        onClick={() => setAddedToCart(false)}
+                        position="absolute"
+                        right="8px"
+                        top="8px"
+                    />
+                </Alert>
+            )}
 
       <Image src={`/${image}`} alt={name} />
 
-      <Box p="5">
-        <Heading as="h4" size="md">
-          {name}
-        </Heading>
-        <p> Price: {price}</p>
-        <AddToCart handleClick={handleAddToCartClick} product={id} >Add To Cart</AddToCart>
-      </Box>
-    </Box>
-  );
+            <Box p="5">
```

Comment:
polecam wtyczke trailing spaces, żeby nie wrzucać białych znaków

Reply:
Maja to chyba nie białe znaki tylko ilość znaków na wcięcie. Ja mam ustawione 2 a piotrekgie chyba 5... U mnie formatuje Prettier. Może zaproponuj wspólne ustawienia?

Reply:
Piotrek,  w sumie możliwe że github to tak zaznacza :) na co dzień używam gerrita i on tak zaznacza właśnie białe znaki, dlatego zwróciłam uwagę. Też korzystam z Prettiera, defaultowe ustawienia, może niech to będą nasze wspólne?

Reply:
Tak u mnie mam tab-a ustawionego na 4 spacje (ustawienia z mojej pracy) - zmieniłem juz u siebie na plikach js na dwie spacje.